paparazzi-gradle-plugin/src/test/projects/compose/src/test/java/app/cash/paparazzi/plugin/test/LoggerLeakTest.kt

```diff
@@ -37,5 +37,7 @@ class LoggerLeakTest {
   }
 
   @Composable
-  private fun ComposeContent() { throw Exception() }
+  private fun ComposeContent() {
+    throw Exception()
+  }
```

Comment:
nit: is this ktlint suggestion or preference?

Reply:
https://github.com/cashapp/paparazzi/pull/1409/commits/897198a85c16b702c13ee1606ddaa9a665db24cc